internal/presenter/packages/spdx_json_presenter.go

```diff
@@ -41,14 +44,19 @@ func (pres *SPDXJsonPresenter) Present(output io.Writer) error {
 
 // newSPDXJsonDocument creates and populates a new JSON document struct that follows the SPDX 2.2 spec from the given cataloging results.
 func newSPDXJsonDocument(catalog *pkg.Catalog, srcMetadata source.Metadata) spdx22.Document {
-	var name string
+	uniqueID := uuid.Must(uuid.NewRandom())
+
+	var name, identifier string
 	switch srcMetadata.Scheme {
 	case source.ImageScheme:
 		name = srcMetadata.ImageMetadata.UserInput
+		identifier = fmt.Sprintf("image/%s", uniqueID.String())
```

Comment:
Note: I think this identifier is meant to include the name, e.g. it should be something like: `http://anchore.com/syft/image/alpine-latest-SOMEUUID`. See: [DocumentNamespace spec](https://spdx.github.io/spdx-spec/document-creation-information/#652-intent), should have: `http://[CreatorWebsite]/[pathToSpdx]/[DocumentName]-[UUID]` this was why I rolled it into the same issue as missing DocumentName.

Reply:
Nice! Good to see it so explicitly formated in their docs. This format is pretty easy for images, but one of the review comments I got was that we want it to be similar across directory and image inputs. 

@luhring what are your thoughts given the spec information? Is it ok if we have some different formatting options for image vs directory?

What should the document name be when we scan a directory or OCI directory source?